src/db/migrations/26-add-jagwar-twin-artifacts.ts

```diff
@@ -0,0 +1,40 @@
+import { Knex } from 'knex';
+
+import { IdExtractorTypes, TitleExtractorTypes } from '../../types/fieldExtractor';
+import { NftFactory, NFTContractTypeName, NFTStandard } from '../../types/nft';
+import { MusicPlatformType } from '../../types/platform';
+import { ethereumArtistId } from '../../utils/identifiers';
+import { addNftFactory, removeNftFactory } from '../migration-helpers';
+
+const jagwarTwinAddress = '0xe80c7fee3d41e311e0351244231919e04e699e56';
+
+const ARTIFACTS: NftFactory = {
+  id: '0x84F89f7bAcb20970073Be33F22847e58fbe78992',
+  startingBlock: '15316663',
+  platformId: jagwarTwinAddress,
+  contractType: NFTContractTypeName.default,
+  standard: NFTStandard.ERC721,
+  autoApprove: false,
+  approved: true,
+  typeMetadata: {
+    overrides: {
+      artist: {
+        name: 'Jagwar Twin',
+        artistId: ethereumArtistId(jagwarTwinAddress),
```

Comment:
i think we should duplicate all artist fields here, since there is something wrong with the upsert of artists that i think is overwriting undefined fields. i think jamie's upcoming PR may fix that but lets just duplicate them here to be safe too

```suggestion
        artistId: ethereumArtistId(jagwarTwinAddress),
                avatarUrl: 'https://web3-music-pipeline.mypinata.cloud/ipfs/QmcBb9C69vvJXasxTYFPwpo9WuZv415KkH3wTdoeBMH2hH',
        websiteUrl: 'https://jagwartwin.com/'
```